static/js/tools/timeline_page.tsx

```diff
@@ -125,6 +126,7 @@ class Page extends Component<PagePropType, PageStateType> {
             <div id="search">
               <SearchBar places={this.state.places} />
             </div>
+            {(window.location.hash === "") && <Info/>}
```

Comment:
check this.state.places.length === 0? There is the case when stats var is selected but place are removed

Reply:
done.